src/cable/udp/cable_udp_obj.cpp

```diff
@@ -82,20 +99,83 @@ void c_cable_udp::async_receive_from(unsigned char *const data, size_t size, rea
 			handler(data, bytes_transferred, std::move(source_addr_cable));
 		} // lambda
 	);
+	*/
+}
+
+bool c_cable_udp::has_separate_rw() const noexcept { ///< do we use separate read and write socket, or is this the same socket
+	switch (m_multisocket_kind) {
+		case (t_multisocket_kind::one_rw): return true; break;
+		case (t_multisocket_kind::separate_rw): return false; break;
```

Comment:
maybe not needed since below we anyway assert in case of not matching either of this values; but added so that code expresses that is what developer intent